zipkin-recorder.go

```diff
@@ -117,10 +117,9 @@ func (r *Recorder) RecordSpan(sp RawSpan) {
 	}
 
 	for _, spLog := range sp.Logs {
-		if spLog.Timestamp.IsZero() {
-			spLog.Timestamp = time.Now()
-		}
-		annotate(span, spLog.Timestamp, spLog.Event, r.endpoint)
+		// XXX: decide how to represent spLog.Fields as an annotation
```

Comment:
@basvanbeek @adriancole this is the only interesting bit... how much / how little of the key:value data do you want to represent as timestamped zipkin Span annotations? If you tell me what you'd like this to be, I'm happy to write up an `Encoder` impl, it'll just take a few minutes.